src/app/services/app-config/appConfigMock.service.ts

```diff
@@ -62,6 +62,44 @@ export class MockAppConfigService {
               }
             ]
           }
+        ],
+        cms: [
```

Comment:
turn into hash/map so something like `cms.credits.url` is possible

Reply:
I'm not sure if creating a hash/map is possible as currently the deployment environment where these values come from is a `.json` file. We could however change it fairly easily into a `.ts` file.

Reply:
sure it is. Not a literal Hash type, but dictionary/hash/map are general terms for key-value based collection... aka an object in JS

in json:

```
"cms" : {
  "credits": {
    "url": "...
  }
}
```